GalaxyZooTouchTable/GalaxyZooTouchTable/Views/SpaceView.xaml.cs

```diff
@@ -7,9 +13,117 @@ namespace GalaxyZooTouchTable.Views
+            DoubleAnimation animateAsideY = new DoubleAnimation(transitionPoint.Y, TimeSpan.FromSeconds(0.25));
+            animateAsideX.BeginTime = TimeSpan.FromSeconds(0.5);
+            animateAsideY.BeginTime = TimeSpan.FromSeconds(0.5);
+            moveAside.BeginAnimation(TranslateTransform.XProperty, animateAsideX);
+            moveAside.BeginAnimation(TranslateTransform.YProperty, animateAsideY);
+
+            TranslateTransform startAside = new TranslateTransform(transitionPoint.X * -1, transitionPoint.Y * -1);
+            TranslateTransform moveInto = new TranslateTransform();
+            DoubleAnimation animateIntoX = new DoubleAnimation(transitionPoint.X, TimeSpan.FromSeconds(0.25));
+            DoubleAnimation animateIntoY = new DoubleAnimation(transitionPoint.Y, TimeSpan.FromSeconds(0.25));
+            animateIntoX.BeginTime = TimeSpan.FromSeconds(0.75);
+            animateIntoY.BeginTime = TimeSpan.FromSeconds(0.75);
+            moveInto.BeginAnimation(TranslateTransform.XProperty, animateIntoX);
+            moveInto.BeginAnimation(TranslateTransform.YProperty, animateIntoY);
+
+            previousCutoutBorderGroup.Children.Add(moveAside);
+            currentCutoutBorderGroup.Children.Add(startAside);
+            currentCutoutBorderGroup.Children.Add(moveInto);
+
+            PreviousCutout.RenderTransform = previousCutoutBorderGroup;
+            CurrentCutout.RenderTransform = currentCutoutBorderGroup;
+        }
+
+        void PerformScaleTransforms()
+        {
+            TransformGroup previousCutoutImageGroup = new TransformGroup();
+            TransformGroup currentCutoutImageGroup = new TransformGroup();
+
+            ScaleTransform scaleSmall = new ScaleTransform();
+            DoubleAnimation animateScaleSmall = new DoubleAnimation(1, 0.5, TimeSpan.FromSeconds(0.25));
+            animateScaleSmall.BeginTime = TimeSpan.FromSeconds(0.25);
+            scaleSmall.BeginAnimation(ScaleTransform.ScaleXProperty, animateScaleSmall);
+            scaleSmall.BeginAnimation(ScaleTransform.ScaleYProperty, animateScaleSmall);
+
+            ScaleTransform scaleLarge = new ScaleTransform();
+            DoubleAnimation animateScaleLarge = new DoubleAnimation(1, 2, TimeSpan.FromSeconds(0.25));
+            animateScaleLarge.BeginTime = TimeSpan.FromSeconds(1);
+            scaleLarge.BeginAnimation(ScaleTransform.ScaleXProperty, animateScaleLarge);
+            scaleLarge.BeginAnimation(ScaleTransform.ScaleYProperty, animateScaleLarge);
+
+            currentCutoutImageGroup.Children.Add(scaleSmall);
+            currentCutoutImageGroup.Children.Add(scaleLarge);
+
+            previousCutoutImageGroup.Children.Add(scaleSmall);
+            previousCutoutImageGroup.Children.Add(scaleLarge);
+
+            CurrentCutoutImage.RenderTransform = currentCutoutImageGroup;
+            PreviousCutoutImage.RenderTransform = previousCutoutImageGroup;
+        }
+
+        void PerformGalaxyFade()
```

Comment:
We can probably just say "PerformFade" as this is namespaced to the SpaceView, but of course that's extremely minor.

Reply:
Makes sense.